silverback/cluster/client.py

```diff
@@ -170,8 +174,8 @@ def start(self):
         response = self.cluster.put(f"/bots/{self.id}", json=dict(name=self.name))
         handle_error_with_response(response)
 
-    @computed_field  # type: ignore[prop-decorator]
     @property
+    @computed_field
```

Comment:
issue: I think this actually breaks things. The `computed_field` decorator must come first.

Reply:
Made the change because mypy was angry about it. I can revert, I'm not sure why your type ignore was no longer working

Reply:
Funny. Pyright recognizes it fine (or allows it anyway), but yeah, mypy hates this for whatever reason.